src/runtime-rs/crates/resource/src/cgroups/mod.rs

```diff
@@ -109,7 +113,22 @@ impl CgroupsResource {
     /// overhead_cgroup_manager to the parent and then delete the cgroups.
     pub async fn delete(&self) -> Result<()> {
         for cg_pid in self.cgroup_manager.tasks() {
-            self.cgroup_manager.remove_task(cg_pid)?;
+            // For now, we can't guarantee that the thread in cgroup_manager does still
+            // exist. Once it exit, we should ignor that error returned by remove_task
+            // to let it go.
+            if let Err(error) = self.cgroup_manager.remove_task(cg_pid) {
+                match error.source() {
+                    Some(err) => match err.downcast_ref::<io::Error>() {
+                        Some(e) => {
+                            if e.raw_os_error() != Some(OS_ERROR_NO_SUCH_PROCESS) {
```

Comment:
Can we add a ErrorKind in cgroup-rs?

Reply:
Hi @YushuoEdge -, What ErrorKind do you mean? no such process error?

Reply:
Oh, true, NoSuchProcess should not distinguished in `remove_task`. The current approach to handle this error may be the best solution！

Reply:
Thanks @YushuoEdge -, I really love this change, but I can't reslove the compile error😞. As this PR is emergency, we can improve it later.

Reply:
OK！